frontend/catalyst/jax_primitives.py

```diff
@@ -917,13 +917,21 @@ def _qinsert_lowering(
 # gphase
 #
 @gphase_p.def_abstract_eval
-def _gphase_abstract_eval(*qubits_or_params, ctrl_len=0, adjoint=False):
+def _gphase_abstract_eval(
+    *qubits_or_params,
+    ctrl_len=0,
+    adjoint=False,
+    static_params=None,
+):
     # The signature here is: (using * to denote zero or more)
     # param, ctrl_qubits*, ctrl_values*
     # since gphase has no target qubits.
-    param = qubits_or_params[0]
+    if not static_params:
```

Comment:
Could the static param here be an integer/float of value 0? Always compare identity with None by default unless you explicitly want to consider other Bool-castable types as well. 
```suggestion
    if static_params is None:
```

Reply:
same would apply below I think

Reply:
I think static param is always a list, but you are definitely right 💯